GoldenPC.yml

```diff
@@ -171,6 +171,18 @@ services:
         command: "nice -n -8 /usr/bin/opendlv-device-cangw-rhino --can=can0 --cid=101 --id=1"
 #        command: "nice -n -8 /usr/bin/opendlv-device-cangw-rhino --can=can0 --enablethrottle --enablebrake --enablesteering --cid=101 --id=1"
 
+#    opendlv-device-can-raw:
```

Comment:
For Eagle, we are using a different version than this that is not using a 8byte string but a uint64 type for a more efficient storage; furthermore, the new version supports multiple CAN channels. I need to merge the new version to master and we need to test the replay for that type of data as well before we can roll-out this part for general use.